cfn-signal/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+eval `ec2cluster`;
```

Comment:
because it does nothing. The result of evaluation is `ec2cluster`. Unless I'm missing something.